### PR TITLE
Developer highlight

### PR DESCRIPTION
**Pull request by mcdennem**

### CODE REVIEW

File: config/bolt/contenttypes.yaml [hunk 1]
```diff
@@ -96,6 +96,34 @@ news:
...
+            type: text
+        twiiter_link:
+            type: text
+        linkedin_link:
+            type: text
+        website_link:
+            type: text
+        interview:
+            type: redactor
+            default: "<p>Tell us a little bit about yourself</p>
```
Comment (I-Valchev) [inline config/bolt/contenttypes.yaml:119]: Love this! Great idea, @mcdennem 👍

File: public/theme/boltcms-2021/partials/_developer_highlight.twig [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<div class="dev-spot">
+    <span>#InTheSpotlight</span>
+
+    <div class="person">
+        <img width="120" height="120" src="{{ record.photo }}" alt="Photo"/>
+        <span class="details">
+                        <h3>{{ record.contributor }}</h3>
```
Comment (I-Valchev) [inline public/theme/boltcms-2021/partials/_developer_highlight.twig:7]: Can you format these? :-)

Comment (mcdennem) [reply public/theme/boltcms-2021/partials/_developer_highlight.twig]: Yes, done!

**Review by I-Valchev — APPROVED** [review]
👍 Nice, great work @mcdennem! Let's use it first for the 5.0 release 😇

**Pull request merged**